otel/otel-release-notes.adoc

```diff
@@ -29,6 +29,33 @@ Data enrichment and processing:: Before forwarding data to analysis tools, the C
 
 Multi-backend receiving and exporting:: The Collector can receive and send data to multiple monitoring and analysis platforms simultaneously.
 
+[id="otel-3-1-rn"]
+== {OTELName} 3.1
+
+{OTELName} 3.1 is based on link:https://opentelemetry.io/[OpenTelemetry] 0.93.0.
+
+[id="new-features-and-enhancements_otel-3-1-rn"]
+=== New features and enhancements
+
+This update introduces the following enhancements:
+
+* Add support for target allocator in the OpenTelemetry collector. The target allocator is an optional component of the operator that shards Prometheus receiver scrape targets across deployed fleet of OpenTelemetry collector. The target allocator as well provides integration with Prometheus `PodMonitor` and `ServiceMonitor` custom resources.
+
+[id="removal-notice_otel-3-1-rn"]
+=== Removal notice
+
+* Nothing
+
+[id="bug-fixes_otel-3-1-rn"]
+=== Bug fixes
+
+* Nothing
+
+[id="known-issues_otel-3-1-rn"]
+=== Known issues
+
+* Nothing
+
 [id="otel-3-0-rn"]
 == {OTELName} {DTProductVersion}
```

Comment:
@max-cx note that once you change DTProductVersion to the 0.93.0 the RN for 3.0 will need to be changed to accommodate the change.